### PR TITLE
Maosen

### PR DESCRIPTION
**Pull request by Milozms**
Fix problem: "TypeError: a bytes-like object is required, not 'str' " in data/source/KBP/generateBClusterInput.py;
Fix wrong path in feature_extraction.sh;
Add requirements installation in feature_extraction.sh;
Add CoreNLP server running instruction in README.md;

### CODE REVIEW

File: data/source/KBP/generateBClusterInput.py [hunk 1]
```diff
@@ -10,7 +10,8 @@
 writtenSents = set()
 for line in file.readlines():
     sent = json.loads(line)
-    sentText = unicodedata.normalize('NFKD', sent['sentText']).encode('ascii','ignore').rstrip('\n').rstrip('\r')
+    # sentText = unicodedata.normalize('NFKD', sent['sentText']).encode('ascii','ignore').rstrip('\n').rstrip('\r')
+    sentText = sent['sentText'].rstrip('\n').rstrip('\r')
```
Comment (cherry979988) [inline data/source/KBP/generateBClusterInput.py]: Hi! I think we need the same modification in folder NYT.

**Comment by shanzhenren** [comment]
Thanks! but I'm wondering in what cases the encoding will generate errors? It seems to me the encoding is still needed as there might be unicode encoding issues in some datasets?
@cherry979988 thanks for checking. @Milozms pls make changes to the script in the other dataset or you can move it to the `./brown_clustering.sh` script I guess

**Comment by shanzhenren** [comment]
Ok, could you integrate the code into `./brown_clustering.sh` and update the PR? I will merge then.

**Comment by Milozms** [comment]
Okay!

**Pull request closed without merging**